lib/elixir/pages/Guards.md

```diff
@@ -191,4 +191,33 @@ def foo(_other) do
 end
 ```
 
-The two forms are exactly the same semantically but there are cases where the latter one may be more aesthetically pleasing.
+For most cases the two forms are exactly the same, there exists a subtle difference in case of failing guards, as discussed in the section above.
```

Comment:
The grammar is a bit off. I think this is clearer. 

`For most cases, the two forms are exactly the same. However, there exists a subtle difference in the case of failing guards, as discussed in the section above.`